### PR TITLE
[#14] Refactor docker-compose.yaml and related files

### PR DESCRIPTION
**Pull request by LucaPrete**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,22 +1,35 @@
-version: '3.0'
+version: '3.3'
 
 services:
-  server:
-    build: .
-    user: "$UID"
```
Comment (ioggstream) [inline docker-compose.yaml:6]: afaik if you remove this line  the container will chown `development_demo`

**Review by ioggstream — CHANGES_REQUESTED** [review]
I'll leave `user`

File: docker-compose.yaml (hunk 1 above)
Comment (LucaPrete) [reply docker-compose.yaml:6]: @ioggstream it will use the default container username. I don't see any issue with it.

**Review by sebbalex — APPROVED** [review]
👍

**Pull request merged**